sdetools/modules/import_veracode/veracode_integrator.py

```
@@ -11,15 +18,40 @@
 
 DEFAULT_MAPPING_FILE = os.path.join(media_path, 'veracode', 'sde_veracode_map.xml')
 
+class VeracodeAPI(RESTBase):
+    """ Base plugin for Veracode """
+
+    def __init__(self, config):
+        super(VeracodeAPI, self).__init__('analysis', 'Veracode', config)
+
+    def parse_response(self, result, headers):
+        # Override to allow xml result
+        try:
+            result = minidom.parseString(result)
+        except Exception:
+            # This means that the result doesn't have XML, not an error
```

On second thought it might be an error. It's just the case when the project named in the config files does not exist and so the the api call returns a 404. This way it wasn't crashing on a non-existent project, but I think you're right. It should throw an exception instead.